frontend/service.go

```diff
@@ -466,6 +466,11 @@ func (s *lwdStreamer) GetMempoolTx(exclude *walletrpc.Exclude, resp walletrpc.Co
 			}
 			newmempoolMap[txidstr] = &walletrpc.CompactTx{}
 			if tx.HasShieldedElements() {
+				txidBytes, err := hex.DecodeString(txidstr)
```

Comment:
In a future PR, we could make `newmempoolMap` more space-efficient by changing its key type to slice of bytes, rather than string. Then we wouldn't have to decode back to bytes here (but we would have to encode to string elsewhere). It's not too bad as it currently is, however, because the mempool typically doesn't have many entries.